### ISSUE TITLE
[Screenshots] Container Network Interface (CNI) under 'How it Works See It in Action' section

### ISSUE THREAD

**Issue by saurabh100ni** [layer5io/layer5#4874]
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/container-network-interface-cni) has only two old screenshots of the designer and visualizer in the 'How it Works See It in Action' section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [Container Network Interface (CNI)](https://layer5.io/cloud-native-management/meshery/integrations/container-network-interface-cni) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by SagarSingh2003** [comment]
I am a beginner , but I would like to work on this issue! , may I?

**Comment by saurabh100ni** [comment]
Sure thing Sagar, Welcome to the community 😎🚀
Ps: you can explore similar PRs for reference.

**Comment by SagarSingh2003** [comment]
> Sure thing Sagar, Welcome to the community 😎🚀 Ps: you can explore similar PRs for reference.

thank you @saurabh100ni , for welcoming me and being so kind ! 

I have set up meshery on my local env , but i am not getting access to the playground!! 

![image](https://github.com/layer5io/layer5/assets/129133613/09fcfd85-1eb4-4bcc-acbf-dbe86d4b7e8a)

can you please tell me if I am doing something wrong? and what i should do next?

**Comment by saurabh100ni** [comment]
@Chadha93 can you take care of access?

**Comment by SagarSingh2003** [comment]
> @Chadha93 can we take care of access?

thank you @saurabh100ni , i've got the access of playground , but i am not able to understand meshmap at all 😔 !! What should I do? can you provide me some resources to learn about it ?

**Comment by abhijeetgauravm** [comment]
> > @Chadha93 can we take care of access?
> 
> thank you @saurabh100ni , i've got the access of playground , but i am not able to understand meshmap at all 😔 !! What should I do? can you provide me some resources to learn about it ?

hey @SagarSingh2003 u can go through this [docs](https://docs.meshery.io/extensions/meshmap) for better understanding of meshmap.

**Comment by SagarSingh2003** [comment]
@abhijeetgauravm @saurabh100ni  i read the docs and have understood how meshmap works , but due to lack of knowledge on CNI and Kubernetes , I am not able to do the design part , do you have something that I can take reference from ?

**Comment by Harsha-10** [comment]
will this pull request get merged as this is labelled with hacktoberfest?

**Comment by saurabh100ni** [comment]
After the review, they do get merged @Harsha-10

**Comment by Harsha-10** [comment]
Okay, thanks for the info

**Comment by saurabh100ni** [comment]
@SagarSingh2003 there are many issues open under Hacktoberfest or you can also take good first issues, see you there 😉

**Comment by Harsha-10** [comment]
Yeah sure 😄

**Comment by SagarSingh2003** [comment]
> @SagarSingh2003 there are many issues open under Hacktoberfest or you can also take good first issues, see you there 😉

thank you @saurabh100ni, i will try to look into other issues then , i've unassigned myself from this one

**Comment by saurabh100ni** [comment]
Alright @SagarSingh2003 ^^
 @Harsha-10 would you wanna work on this issue?

**Comment by Harsha-10** [comment]
I have a small doubt on these issues

**Comment by Harsha-10** [comment]
Do I need to attach the screenshot of actual design integration or can I use some random design in meshery and attach the screenshot?

**Comment by Harsha-10** [comment]
Like I have checked 2 similar issues and both have same screenshots of a single design

**Comment by Harsha-10** [comment]
okay,But i don't know how to make designs using the integration....

**Comment by sandramsc** [comment]
Hi @Harsha-10, in addition to what @saurabh100ni has already stated, consider reading this [message on Slack](https://layer5io.slack.com/archives/CDM0ACDM5/p1697115088411519). I also propose the conversation continues on Slack, to potentially get assistance from the wider community as well as to reduce notifications for maintainers on here, so they can focus on more essential tasks. Any further clarifying questions, please feel free to ask.

**Comment by Harsha-10** [comment]
Yeah, i can understand. I have got an overview after checking the post, thanks for the support @saurabh100ni @sandramsc

**Comment by abhijeetgauravm** [comment]
> okay,But i don't know how to make designs using the integration....

@Harsha-10 regarding design you don't need to create your own, you can use published design and clone them and take screenshot of them (You can do that in meshmap)

**Comment by Harsha-10** [comment]
Yeah got it thanks @abhijeetgauravm

**Comment by sandramsc** [comment]
Hi,  @Harsha-10 how are you getting along with the solution? Let us know if there is an update on progress and remember to utilize the discussion forum might you have any clarifying questions.

**Comment by Harsha-10** [comment]
@sandramsc I'm not working on the issue, just got the info then and worked on similar issues and i have not been assigned

**Comment by sandramsc** [comment]
> @sandramsc I'm not working on the issue, just got the info then and worked on similar issues and I have not been assigned

That's right, double checked, let us know if the information provided was helpful in getting you started with the solution for the similar issue and consider tagging this issue for context might you have any further questions.

**Comment by ruthwikchikoti** [comment]
hey @sandramsc  can you assign me this issue .

**Comment by narasimha-1511** [comment]
here you go @ruthwikchikoti 🚀
